flink-connectors/flink-connector-elasticsearch6/src/main/java/org/apache/flink/streaming/connectors/elasticsearch6/ElasticsearchSink.java

```diff
@@ -97,6 +97,15 @@ public Builder(List<HttpHost> httpHosts, ElasticsearchSinkFunction<T> elasticsea
 			this.elasticsearchSinkFunction = Preconditions.checkNotNull(elasticsearchSinkFunction);
 		}
 
+		/**
+		 * Sets a map of strings that configures the bulk flushing behaviour.
+		 *
+		 * @param bulkRequestsConfig user configuration to configure bulk flushing behaviour.
+		 */
+		public void setBulkRequestsConfig(Map<String, String> bulkRequestsConfig) {
+			this.bulkRequestsConfig = bulkRequestsConfig;
```

Comment:
Check for null. But do we actually need to expose the raw map if it is not used in ES 6 and later?

Reply:
My thought was just that it's more or less a convenience for users. In some scenarios, having a properties like setter would be easier to use. I'm completely ok with removing it, though.